tests/_support/AcceptanceTester.php

```diff
@@ -342,7 +342,7 @@ public function optOutOfSubscription() {
    */
   public function selectPaymentMethod($method = 'cod') {
     $I = $this;
-    $I->scrollTo('#payment_method_' . $method);
+    $I->scrollTo('#payment_method_' . $method, 0, -40);
```

Comment:
This magic number needs to have some comment. Why do we need it? Why exactly 40?